src/app/header/header.component.ts

```diff
@@ -1,4 +1,6 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Input } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
```

Comment:
Remove unused imports everywhere in the app

Reply:
unused imports removed everywhere in the app